crates/graphql/src/schema/objects/nft.rs

```diff
@@ -235,7 +236,8 @@ impl TryFrom<models::NftActivity> for NftActivity {
         Ok(Self {
             id,
             metadata: metadata.into(),
-            auction_house: auction_house.into(),
+            auction_house: auction_house.clone().into(),
+            marketplace_address: auction_house,
```

Comment:
hmm this doesn't sit right. we should update the NftActivity query to return martplace_address.